rskj-core/src/test/java/co/rsk/peg/federation/NonStandardErpFederationsTest.java

```diff
@@ -1005,7 +997,7 @@ private void validateErpRedeemScript(
         assertEquals(ScriptOpCodes.getOpCode(String.valueOf(n)), script[index++]);
 
         assertEquals(ScriptOpCodes.OP_ENDIF, script[index++]);
-        assertEquals(Integer.valueOf(ScriptOpCodes.OP_CHECKMULTISIG).byteValue(), script[index++]);
+        assertEquals(Integer.valueOf(ScriptOpCodes.OP_CHECKMULTISIG).byteValue(), script[index]);
```

Comment:
why this change?

Reply:
No need to increase the value of `index` at that point. It's the last time it's used